### ISSUE TITLE
Screenreader nvda does not read lines correctly

### ISSUE THREAD

**Issue by gFrincu** [MiKTeX/miktex#1297]
Hello.
I am a blind person and I use nvda screenreader to work with LaTeX, first of all, I am sorry because I don't understand much of how it works, I can't understand well what's the difference between installing Miktex or texworks instead.
Okay, having said this my problem is that I changed my laptop and I installed miktex and nvda (the screenreader).  I wanted to work in a LaTeX file and when I go through a blank line, it reads the last line that has any content written but it does not read blank line, and makes the navigation a bit annoying.
Moreover, if the line is very long, it reads it all by one time, and does not separate it as it is written.
I will make an issue repport to nvda too, but I wanted to ask if maybe something from the framework or the window view changed. 
In my new laptop I have the latest version, in my old one, it says this:

About TeXworks  alerta  TeXworks is a simple environment for editing, typesetting, and previewing TeX documents.
© 2007-2017 Jonathan Kew, Stefan Löffler, Charlie Sharpsteen
Version 0.6.2 (MiKTeX 2.9.6600 64-bit) [r.7ecce17, 29/04/2017 18:23]
Distributed under the GNU General Public License, version 2 or (at your option) any later version.
Qt application framework v5.9.2 by The Qt Company.
Poppler PDF rendering library by Kristian Høgsberg, Albert Astals Cid and others.
Hunspell spell checker by László Németh.
Concept and resources from TeXShop by Richard Koch.
SyncTeX technology by Jérôme Laurens.
Some icons used are from the Tango Desktop Project.

If you want, I can try to attach some audio files to show you the difference.

[miktex-report.txt](https://github.com/MiKTeX/miktex/files/10853566/miktex-report.txt)

Sorry for my English,
Thanks
Georgiana
